### PR TITLE
chore(add) :styled components Model, Text and Events [COM-40]

### PR DESCRIPTION
**Pull request by terereyna**
Implement styled components to Events, Model and Text.

### CODE REVIEW

File: src/components/Events/Events.js [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
   return (
-    <div className="container-event">
-      <div className="cols">
+    <ContainerEvent>
+      <ColsEvent className="cols-event">
         <div className="event">
-          <h1>Los próx. eventos del Reap Society</h1>
-          <br />
-          <p>
+          <H1Event className="h1-event">Los próx. eventos del Reap Society</H1Event><br />
```
Comment (gonzajo) [inline src/components/Events/Events.js]:
```suggestion
          <TitleEvent className="h1-event">Los próx. eventos del Reap Society</TitleEvent><br />
```

File: src/components/Events/Events.css.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    font-size: 60px;
+    text-transform: uppercase;
+    color: #d4d6d4;
+    font-weight: bold;
+    text-align: left;
+    margin: 0 480px 0 0;
+`;
+
+export const EventsP = styled.p`
+`;
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]: no need to create this if the are no styles

File: src/components/Events/Events.css.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+export const Pevent = styled.p`
+    .event  .p-event{
+    font-size: 15px;
+    margin: 5px;
+    }
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]:
```suggestion
    font-size: 15px;
    margin: 5px;
```

File: src/components/Events/Events.css.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    font-weight: 600; 
+    border: 3px solid #02F181;
+    background-color: #02F181;
+    border-radius: 35px;  
+    color: black;  
+    border-width: 3px;
+    font-size: 15px; 
+    .event .button-event{
+    margin: 0 300px 10px 20px;
+    }
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]:
```suggestion
    margin: 0 300px 10px 20px;
```

File: src/components/Events/Events.css.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    color: #d4d6d4;
+    `;
+
+export const ColsEvent = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px;   
+    div{
+    flex: 2;
+    }
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]:
```suggestion
    flex: 2;
```

File: src/components/Events/Events.js [hunk 6]
```diff
@@ -1,29 +1,54 @@
...
+          </p><br/>
+          </div>
+          <button className="button-event">Agrega tu evento</button>
+        </div>
+        <div className="col2">
+          <p className="p-event2">No hay próximos eventos.</p>
+        </div>
+      </div>
+    </ContainerEvent>
+
```
Comment (gonzajo) [inline src/components/Events/Events.js]: remove this comment, if there are no use of it

File: src/components/Text/Text.css.js [hunk 7]
```diff
@@ -1,8 +1,10 @@
-.main-container2 {
+import styled from 'styled-components';
+
+export const MainContainer2 = styled.div`
```
Comment (gonzajo) [inline src/components/Text/Text.css.js]:
```suggestion
export const Container = styled.div`
```

File: src/components/Text/Text.css.js [hunk 8]
```diff
@@ -1,8 +1,10 @@
...
     background-color: #DCDBDB;
-} 
+`;
 
-.main-container2 h1 {
+export const Texth1 = styled.h1`
```
Comment (gonzajo) [inline src/components/Text/Text.css.js]:
```suggestion
export const Title = styled.h1`
```

File: src/components/Events/Events.css.js [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+    color: #d4d6d4;
+    text-align: left;
+    padding: 0 450px 0 0;
+    font-family: Arial;
+`;
+
+export const Pevent2 = styled.p`
+    font-size: 15px;
+    width: left;
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]: width with left? i think this is wrong...

File: src/components/Events/Events.css.js [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+    font-size: 15px;
+    margin: 5px;
+    }
+    color: #d4d6d4;
+    text-align: left;
+    padding: 0 450px 0 0;
+    font-family: Arial;
+`;
+
+export const Pevent2 = styled.p`
```
Comment (gonzajo) [inline src/components/Events/Events.css.js]: the name should be more explicit, it is a subtitle or a paragraph
what is the intention for this component?

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/components/Events/Events.css.js (hunk 10 above)
Comment (terereyna) [reply src/components/Events/Events.css.js]: Is a paragraph. The intention is view the events. It is already correct.

**Review by gonzajo — APPROVED** [review]

**Comment by gonzajo** [comment]
@tere-reyna98 resolve the merge conflict plis

**Pull request closed without merging**